### PR TITLE
Deref user tournaments

### PR DESCRIPTION
**Pull request by ShiroUsagi-san**
Adding fields to populate user dashboard with less API requests

### CODE REVIEW

**Comment by Lymkwi** [comment]
I'll queue this after #93 and #94 are merged because I don't want to change tournaments from two places at once and not know which one broke the test suite :|

File: insalan/tournament/views.py [hunk 1]
```diff
@@ -189,6 +188,67 @@ def get(self, request, primary_key: int):
...
+    permission_classes = [permissions.IsAuthenticated & ReadOnly]
+
+    def get(self, request):
+        user: User = request.user
+
+        if user is None:
+            raise PermissionDenied()
+
+        # retrieve inscription as Player
+        player_inscriptions = Player.objects.filter(user=user)
```
Comment (Lugrim) [inline insalan/tournament/views.py:206]: This is no registration, this is a player, please keep a consistent name for variables

File: insalan/tournament/views.py [hunk 2]
```diff
@@ -189,6 +188,67 @@ def get(self, request, primary_key: int):
...
+        # serialize it
+        player_inscriptions = serializers.PlayerSerializer(
```
Comment (Lugrim) [inline insalan/tournament/views.py:208]: The English word for "inscription" is "registration". Please change it and afterwards to keep a consistent and understandable code base.

**Review by Lugrim — CHANGES_REQUESTED** [review]
It looks like it's working, however the code base (especially the variable names) should be reworked to keep it understandable.

Also, tests should be added, it may be done later though (please open an issue and mention it here to keep a trace of what's happening)

File: insalan/tournament/views.py (hunk 1 above)
Comment (Lymkwi) [reply insalan/tournament/views.py:206]: at the very least please call it something like `player_registration`, I'd be very upset if people inscribed things on me

**Review by Lugrim — APPROVED** [review]
Great, better than what I expected!

**Pull request merged**